secretservice/secretservice_linux_test.go

```diff
@@ -16,10 +17,39 @@ func TestSecretServiceHelper(t *testing.T) {
 	}
 
 	helper := Secretservice{}
+
+	// Check how many docker credentials we have when starting the test
+	old_auths, err := helper.List()
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	// If any docker credentials with the tests values we are providing, we
+	// remove them as they probably come from a previous failed test
+	if len(old_auths) >= 1 {
```

Comment:
Not sure if it's required here. The for loop just won't do a thing if the length of `old_auths` is `0` 👼